src/pre_process/m_patches.fpp

```diff
@@ -1410,70 +1438,15 @@ contains
         !! @param patch_id_fp Array to track patch ids
         !! @param q_prim_vf Array of primitive variables
     subroutine s_2D_analytical(patch_id, patch_id_fp, q_prim_vf)
-
         integer, intent(in) :: patch_id
         integer, dimension(0:m, 0:n, 0:p), intent(inout) :: patch_id_fp
         type(scalar_field), dimension(1:sys_size), intent(inout) :: q_prim_vf
 
-        integer :: i, j, k !< generic loop iterators
+        print *, "WARNING :: The use of the 2D analytical patch geometry has &
```

Comment:
oh i see, you are going to make this for actual analytic semantics. but what if the 2D analytical patch isn't a rectangle (say, it could be a circle)? I guess you could put that as a parameter to the subroutine/input file.

Reply:
btw all print statements need to have ` if (proc_rank == 0 )` before them so they don't print out 24 (or, even worse, 30K) lines that say the same thing, one for each MPI rank.